### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by donizer**
- [DEMO LINK](https://donizer.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://donizer.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+@font-face {
+  font-family: "AvenirBook";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:15]: image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  width: 11px;
+  padding: 0 9px;
+}
+
+.search__input {
+  all: unset;
+
+  width: 100%;
+
+  font-family: AvenirBook, Arial, Helvetica, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:61]: you can just apply font-family to `html` tag selector and let inputs to inherit it 
```suggestion
  font-family: inherit;
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  font-weight: 400;
+}
+
+.search__input::placeholder {
+  color: var(--text-color);
+  opacity: 1;
+}
+
+.search__input:focus {
+  font-family: AvenirHeavy, Arial, Helvetica, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:73]: same here
```suggestion
  font-family: inherit;
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Good job! Almost done

**Comment by donizer** [comment]
So, what about changes, is new solution fine?

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,39 @@
...
+        alt="icon"
+      >
+      <input
+        type="text"
+        class="search__input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+    <form
```
[the remark this answers is not in the document]
Comment (DarkMistyRoom) [reply src/index.html:32]: Add empty lines between multiline sibling blocks of HTML

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good job! But check this requirement
image
image

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  --text-color: #3d4e61;
+
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
```
Comment (loralevitska) [inline src/style.css:12]: Remove redundant default value. Fix all cases

**Review by loralevitska — APPROVED** [review]
Well done) Check my comments for improvement

**Pull request closed without merging**